src/main.rs

```diff
@@ -21,11 +22,33 @@ async fn main() {
         tracing::warn!("Failed to load .env file: {}", e);
     }
 
+    // Get server host and port from environment variables with fallbacks
+    let host = std::env::var("SERVER_HOST").unwrap_or_else(|_| "127.0.0.1".to_string());
+    let port = std::env::var("SERVER_PORT")
+        .unwrap_or_else(|_| "3000".to_string())
+        .parse::<u16>()
+        .expect("SERVER_PORT must be a valid port number");
+
+    // Check for database configuration
+    let database_url = std::env::var("DATABASE_URL").unwrap_or_else(|_| {
+        let db_user = std::env::var("POSTGRES_USER").unwrap_or_else(|_| "postgres".to_string());
+        let db_pass = std::env::var("POSTGRES_PASSWORD").unwrap_or_else(|_| "postgres".to_string());
+        let db_host = std::env::var("POSTGRES_HOST").unwrap_or_else(|_| "localhost".to_string());
+        let db_port = std::env::var("POSTGRES_PORT").unwrap_or_else(|_| "5432".to_string());
+        let db_name = std::env::var("POSTGRES_DB").unwrap_or_else(|_| "lifeforce".to_string());
+        
+        format!("postgres://{}:{}@{}:{}/{}", db_user, db_pass, db_host, db_port, db_name)
+    });
+    
+    tracing::info!("Using database URL: {}", database_url);
```

Comment:
Logging the connection URL especially at info level may not be ideal especially if the password would be there in plaintext.